src/isar/modules.py

```diff
@@ -150,6 +161,13 @@ def provide_scheduling_utilities(
     ) -> SchedulingUtilities:
         return SchedulingUtilities(queues, mission_planner)
 
+class RobotUtilitiesModule(Module):
```

Comment:
As this new module is introduced, should we consider renaming the `UtilitiesModule` to `SchedulingUtilitiesModule` for specification?